src/shared/utils/__tests__/utils.test.ts

```diff
@@ -4,60 +4,108 @@ import { SchemaId, WrappedDocument } from "../../../shared/@types/document";
 
+const baseIssuerObjectV2: v2.Issuer = {
+  name: "John",
+  identityProof: {
+    type: v2.IdentityProofType.DNSTxt,
+    location: "tradetrust.io",
+  },
+};
+
+const baseV2DocuemntObject = {
+  version: SchemaId.v2,
+  data: {},
+  signature: {
+    type: "SHA3MerkleProof",
+    targetHash: "64b2ed566455d0adbc798a8f824f163d87276dcbd66cacff8a6a4ba28fb800fc",
+    proof: [],
+    merkleRoot: "64b2ed566455d0adbc798a8f824f163d87276dcbd66cacff8a6a4ba28fb800fc",
+  },
+};
+
+const baseV3DocumentObject = {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
+  ],
+  issuer: {
+    name: "name",
+    type: "OpenAttestationIssuer",
+    id: "https://example.com",
+  },
+  issuanceDate: "2010-01-01T19:23:24Z",
+  type: ["VerifiableCredential", "UniversityDegreeCredential", "OpenAttestationCredential"],
+  credentialSubject: {
+    id: "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    degree: {
+      type: "BachelorDegree",
+      name: "Bachelor of Science and Arts",
+    },
+  },
+  openAttestationMetadata: {
+    proof: {
+      value: "0xabcf",
+      type: v3.ProofType.OpenAttestationProofMethod,
+      method: v3.Method.DocumentStore,
+    },
+    identityProof: {
+      identifier: "whatever",
+      type: v2.IdentityProofType.DNSTxt,
+    },
+    template: {
+      url: "https://",
+      name: "",
+      type: v3.TemplateType.EmbeddedRenderer,
+    },
+  },
+  name: "",
+  reference: "",
+  validFrom: "2010-01-01T19:23:24Z",
+};
+
 describe("Util Functions", () => {
   let wrappedV3Document: WrappedDocument<v3.OpenAttestationDocument>;
-  let wrappedTransferableV3Document: WrappedDocument<v3.OpenAttestationDocument>;
   const wrappedV2Document: WrappedDocument<v2.OpenAttestationDocument> = wrapDocument({
     issuers: [
       {
-        name: "John",
-        documentStore: "0xabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd",
-        identityProof: {
-          type: v2.IdentityProofType.DNSTxt,
-          location: "example.com",
-        },
+        ...baseIssuerObjectV2,
+        documentStore: "0x9178F546D3FF57D7A6352bD61B80cCCD46199C2d",
       },
     ],
   });
   const wrappedTransferableV2Document: WrappedDocument<v2.OpenAttestationDocument> = wrapDocument({
     issuers: [
       {
-        name: "John",
+        ...baseIssuerObjectV2,
         tokenRegistry: "0xabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd",
-        identityProof: {
-          type: v2.IdentityProofType.DNSTxt,
-          location: "example.com",
-        },
       },
     ],
   });
+  let v2TransferableDocument: WrappedDocument<any>;
+  let v2VerifiableDocument: WrappedDocument<any>;
+  let v3TransferableDocument: WrappedDocument<v3.OpenAttestationDocument>;
+  let v3VerifiableDocument: WrappedDocument<v3.OpenAttestationDocument>;
+  let v3DidDocument: WrappedDocument<v3.OpenAttestationDocument>;
```

Comment:
follow the `wrapped` prefix convention? 

```suggestion
  let wrappedV2TransferableDocument: WrappedDocument<any>;
  let wrappedV2VerifiableDocument: WrappedDocument<any>;
  let wrappedV3TransferableDocument: WrappedDocument<v3.OpenAttestationDocument>;
  let wrappedV3VerifiableDocument: WrappedDocument<v3.OpenAttestationDocument>;
  let wrappedV3DidDocument: WrappedDocument<v3.OpenAttestationDocument>;
```